### ISSUE TITLE
Screen mess when 

### ISSUE THREAD

**Issue by damogongzi** [juliancwirko/meteor-s-alert#32]
When add materializer package in meteor project and trigger a sAlert, the screen turns to mess for the first time, the rest of the call, works fine.

http://materializecss.com/getting-started.html

compared with themeteorchef:bert, it works fine.

**Comment by juliancwirko** [comment]
Where have you included your `{{> sAlert}}` template? 
Can I see some screenshot or demo url?

**Comment by juliancwirko** [comment]
Also I have tested it on my project with materialize and it works good with:

```
<body>
    {{> sAlert}}
</body>
```

**Comment by damogongzi** [comment]
The code like: 

<head>
	<title>Materialize_sAlert</title>
	
	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
	<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
</head>

<body class="grey lighten-3">
	<h1>Welcome to Meteor!</h1> {{> sAlert}}  {{> hello}} 
</body>



		sAlert.config({
			effect: 'bouncyflip',  //has some UI display issue when this effect is enabled.
			position: 'top-right',
			timeout: 3000,
			html: false,
			onRouteClose: true,
			stack: true,
			offset: 0, // in px - will be added to first alert (bottom or top - depends of the position in config)
			beep: false
				// examples:
				// beep: '/beep.mp3'  // or you can pass an object:
				// beep: {
				//     info: '/beep-info.mp3',
				//     error: '/beep-error.mp3',
				//     success: '/beep-success.mp3',
				//     warning: '/beep-warning.mp3'
				// }
		});

packages:

juliancwirko:s-alert
juliancwirko:s-alert-bouncyflip
materialize:materialize

**Comment by juliancwirko** [comment]
Can you try to put it somewhere in the `<body>{{> sAlert}}</body>` tag? Like it is in the documentation.
Let me know if this work. It is hard to say what is the problem without seeing all css and html code. Bouncyflip effect uses some transformations and when it is used in some custom wrappers the css could not work well, so it is always better to put it in main body tag.

**Comment by damogongzi** [comment]
It is in the body tag as listed above.

<--body class="grey lighten-3">
<--h1>Welcome to Meteor! {{> sAlert}} {{> hello}}
<--/body> 

Please note that I added "<--" just because it could not be displayed properly here. It should be normal html tag.

No extras css used instead of materializer css like: body class="grey lighten-3". I tried to remove it but it does not help to resolve the issue.

check out the url: http://materializesalert.meteor.com/

try to click the "click me" button.

**Comment by juliancwirko** [comment]
Is this something like:

```
<body>
<h1>Welcome {{> sAlert}} {{> hello}}</h1>
</body>
```
??

Try to do something like:

```
<body>
    <h1>Welcome {{> hello}}</h1>
    {{> sAlert}}
</body>
```

sAlert should be placed outside any wrapper.

Also screenshot with this bug will be cery helpful.

**Comment by juliancwirko** [comment]
for me your demo works good. Which browser do you use?

**Comment by damogongzi** [comment]
Yep, you are right. It works good in firefox. I am using chrome. In IE, there is a flash, then show the notification. I think it may be related with css compatibility among browsers.
